### ISSUE TITLE
Scala Common Enrich: externalize referer-parser yml file

### ISSUE THREAD

**Issue by szareiangm** [snowplow/snowplow#3830]
The enhancement that a community member has done for `ua-parser` enrichment, it turned out that it is possible to do the same for `referer-parser` enrichment:

https://github.com/snowplow-referer-parser/referer-parser/blob/99e7e383ccd27c423c0a62eabc1dfb6e10a5a15f/java-scala/src/main/java/com/snowplowanalytics/refererparser/Parser.java#L81

Therefore, we can feed our custom list without updating the pipeline codebase.

**Comment by alexanderdean** [comment]
Yep, that's the plan.

**Comment by szareiangm** [comment]
If I could find some free time this week or the next, will do it. Busy with the other promise for optional enrichments.

**Comment by BenFradet** [comment]
blocked by snowplow-referer-parser/jvm-referer-parser#22

**Comment by szareiangm** [comment]
Hello and sorry for long silence on this topic. 

I noticed that there was a great progress on the Scala library. Also, snowplow-referer-parser/scala-referer-parser#22 is merged into 0.4.0 and there 0.5.0 is out, too. Can I ask what is blocking this ticket to bump to 0.5.0 and externalize the file as part of it?

**Comment by BenFradet** [comment]
Hey Saeed, this is going into R115.
